frontend/model/contracts/shared/giLodash.js

```diff
@@ -78,6 +78,15 @@ export function randomHexString (length: number): string {
   return Array.from(randomBytes(length), byte => (byte % 16).toString(16)).join('')
 }
 
+export function normalizeString (str: string): string {
+  return str
+    // [1]. Normalize strings by replacing both apostrophes and single quotes with a standard character (reference issue: https://github.com/okTurtles/group-income/issues/2479)
+    .replace(/['’]/g, "'")
```

Comment:
I think that this will not catch all possible cases. See https://github.com/okTurtles/group-income/issues/2479#issuecomment-2585274842 for a more general solution.

Reply:
@corrideat  Feel free to work on the issue yourself. I have no objection to it. (I'm actually busy working on another issue and I heard Greg said you are looking for issue to work on) 

just re-assign yourself to that issue and let me know. I will discard this PR.